lib/actions/ssh.coffee

```diff
@@ -115,8 +117,8 @@ module.exports =
 					type: 'list'
 					choices: _.map containers, (container) ->
 						return {
-							name: "#{container.name or 'Untitled'} (#{container.id})"
-							value: container.name
+							name: "#{container.Names[0] or 'Untitled'} (#{container.Id})"
```

Comment:
You should check this on an older device as well, I believe docker changed from `id`/`name` to `Id`/`Name` at some point, so this may not work on older devices anymore